src/llm/servable.cpp

```diff
@@ -207,6 +208,26 @@ std::string wrapTextInServerSideEventMessage(const std::string& text) {
     ss << "data: " << text << "\n\n";
     return ss.str();
 }
+std::string getBodyWithDecodedContent(const ovms::HttpPayload& payload) {
```

Comment:
what is the point of this payload parsing? In a critical performance path, all non essential processing should be removed